.github/workflows/destroy.yml

```diff
@@ -16,6 +16,27 @@ jobs:
         with:
           terraform_version: 0.14.0
 
+      - name: Clear S3 bucket
+        env:
+          CF_USERNAME: ${{ secrets.GOVPAAS_DEV_USERNAME }}
+          CF_PASSWORD: ${{ secrets.GOVPAAS_DEV_PASSWORD }}
+        run: |
+          wget -q -O - https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key | sudo apt-key add -
+          echo "deb https://packages.cloudfoundry.org/debian stable main" | sudo tee /etc/apt/sources.list.d/cloudfoundry-cli.list
+          sudo apt-get update
+          sudo apt-get install cf7-cli
+          cf api https://api.london.cloud.service.gov.uk
+          cf auth
+          cf target -o "dfe" -s "earlycareers-framework-dev"
+          cf create-service-key ecf-csv-review-pr-${{ github.event.number }} ecf-csv-review-pr-${{ github.event.number }}-key -c '{"allow_external_access": true}'
+          CREDS=$(cf service-key ecf-csv-review-pr-${{ github.event.number }} ecf-csv-review-pr-${{ github.event.number }}-key)
+          AWS_ACCESS_KEY_ID=$(echo $CREDS | tail -7 | jq -r '.aws_access_key_id')
+          AWS_SECRET_ACCESS_KEY=$(echo $CREDS | tail -7 | jq -r '.aws_secret_access_key')
+          BUCKET_NAME=$(echo $CREDS | tail -7 | jq -r '.bucket_name')
+          aws s3 rm s3://$BUCKET_NAME --recursive
+          cf delete-service-key ecf-csv-review-pr-${{ github.event.number }} ecf-csv-review-pr-${{ github.event.number }}-key -f
```

Comment:
Is there no way to do it via cf? https://cli.cloudfoundry.org/en-US/v7/delete-service.html

Reply:
https://docs.cloud.service.gov.uk/deploying_services/s3/#:~:text=you%20must%20delete%20all%20objects%20from%20the%20s3%20bucket%20before%20you%20can%20delete%20the%20bucket%20itself

Reply:
Looks like TVS do something different, but I cannot find any documentation on this https://github.com/DFE-Digital/teaching-vacancies/blob/30956313bee65abcc894216a8c915215ba86fd49/terraform/app/modules/paas/main.tf#L30

Reply:
Aha, found it https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket#force_destroy

Reply:
I'm not sure we can use that. They use AWS credentials and an AWS provider and provision the bucket directly. We have do everything through PaaS

Reply:
Well, if it doesn't work out then this approach is fine

Reply:
https://github.com/DFE-Digital/teaching-vacancies/blob/30956313bee65abcc894216a8c915215ba86fd49/terraform/app/modules/paas/versions.tf#L4

Reply:
That won't work for us. We don't have access to AWS in the same way they do